src/archiefbeheercomponent/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -139,183 +139,237 @@ msgstr ""
 msgid "In progress"
 msgstr "In behandeling"
 
-#: archiefvernietigingscomponent/destruction/constants.py:39
+#: archiefbeheercomponent/destruction/constants.py:39
 msgid "Action required"
 msgstr "Actie vereist"
 
-#: archiefvernietigingscomponent/destruction/constants.py:40
+#: archiefbeheercomponent/destruction/constants.py:40
 msgid "Completed"
 msgstr "Verwerkt"
 
-#: archiefvernietigingscomponent/destruction/constants.py:41
+#: archiefbeheercomponent/destruction/constants.py:41
 msgid "All"
 msgstr "Alle"
 
-#: archiefvernietigingscomponent/destruction/constants.py:46
+#: archiefbeheercomponent/destruction/constants.py:46
 msgid "Changes requested"
 msgstr "Wijzigingen vereist"
 
-#: archiefvernietigingscomponent/destruction/constants.py:47
+#: archiefbeheercomponent/destruction/constants.py:47
 msgid "Rejected"
 msgstr "Afgewezen"
 
-#: archiefvernietigingscomponent/destruction/constants.py:48
+#: archiefbeheercomponent/destruction/constants.py:48
 msgid "Approved"
 msgstr "Goedgekeurd"
 
-#: archiefvernietigingscomponent/destruction/constants.py:49
+#: archiefbeheercomponent/destruction/constants.py:49
 msgid "Finished"
 msgstr "Verwerkt"
 
-#: archiefvernietigingscomponent/destruction/models.py:35
-#: archiefvernietigingscomponent/destruction/models.py:254
+#: archiefbeheercomponent/destruction/constants.py:57
+msgid ""
+"Het zaakdossier moet bewaard blijven en op de Archiefactiedatum overgedragen "
+"worden naar een archiefbewaarplaats."
+msgstr ""
+
+#: archiefbeheercomponent/destruction/constants.py:63
+msgid "Het zaakdossier moet op of na de Archiefactiedatum vernietigd worden."
+msgstr ""
+
+#: archiefbeheercomponent/destruction/constants.py:70
+msgid "De zaak cq. het zaakdossier is nog niet als geheel gearchiveerd."
+msgstr ""
+
+#: archiefbeheercomponent/destruction/constants.py:75
+msgid ""
+"De zaak cq. het zaakdossier is als geheel niet-wijzigbaar bewaarbaar gemaakt."
+msgstr ""
+
+#: archiefbeheercomponent/destruction/constants.py:81
+msgid ""
+"De zaak cq. het zaakdossier is als geheel niet-wijzigbaar bewaarbaar gemaakt "
+"maar de vernietigingsdatum kan nog niet bepaald worden."
+msgstr ""
+
+#: archiefbeheercomponent/destruction/constants.py:87
+msgid ""
+"De zaak cq. het zaakdossier is overgebracht naar een archiefbewaarplaats."
+msgstr ""
+
+#: archiefbeheercomponent/destruction/forms.py:177
+msgid ""
+"If '%(create_zaak_label)s' is checked, all the other destruction case "
+"settings need to be configured."
+msgstr ""
+"Als '%(create_zaak_label)s' aangevinkt is, moeten alle andere zaak instellingen "
+"geconfigureerd worden."
+
+#: archiefbeheercomponent/destruction/forms.py:204
+msgid ""
+"If no zaak is created after processing a destruction list, the destruction "
+"report should be downloadable through the app."
+msgstr ""
+"Als er geen zaak wordt aangemaakt na het vernietigen van een lijst, zou "
+"de verklaring van vernietiging via de app kunnen worden gedownload."
```

Comment:
```suggestion
"Als er geen zaak wordt aangemaakt na het vernietigen van een lijst, dan "
"kan de verklaring van vernietiging in deze applicatie worden gedownload."
```

"zou kunnen" sounds a bit weak, like "possibly you could", but it says "should".